performant-software/Neatline
# Removed development assets from the repository

### What does this PR Do?

- adds `views/**/dist/development` back to to .gitignore
- removes those files from the repo, via:
  - `git rm -rf --cached views/**/dist/development`
- adds new production assets to the repo, via:
  - `grunt build` then
  - `grunt freeze`, which does: `compile:min` and `copy-payloads`, 
  - which copies built assets to `views/**/dist/production`

### How to test

- Visit: http://neatline-staging.herokuapp.com/neatline/show/london
- The exhibit should be working properly. This was not the case previously, unless the `APPLICATION_ENV` config var was set to `development`. It's now set to `production`, which is how it should be.
